File: fingertip/src/main/java/com/praszapps/fingertip/SampleUsage.java
package com.praszapps.fingertip;

import android.os.Bundle;
import android.support.annotation.Nullable;
import android.support.v7.app.AppCompatActivity;

import com.praszapps.fingertip.manager.AuthenticationStyle;
import com.praszapps.fingertip.manager.FingertipAuthConfig;
import com.praszapps.fingertip.manager.FingertipAuthenticationResult;
import com.praszapps.fingertip.manager.FingertipAuthenticator;

public class SampleUsage extends AppCompatActivity {
    FingertipAuthConfig config = new FingertipAuthConfig();

    @Override
    protected void onCreate(@Nullable Bundle savedInstanceState) {
        super.onCreate(savedInstanceState);
        config.setAuthenticationStyle(AuthenticationStyle.FINGERPRINT_DIALOG);
        FingertipAuthenticator.INSTANCE.doAuth(config, new FingertipAuthenticationResult() {
            @Override
            public void onFingertipAuthSuccess() {

            }

            @Override
            public void onFingertipAuthFailed(String errorString) {

            }
        });
    }
}

File: fingertip/src/main/java/com/praszapps/fingertip/SampleUsage.kt
package com.praszapps.fingertip

import android.os.Bundle
import android.support.v7.app.AppCompatActivity

import com.praszapps.fingertip.manager.AuthenticationStyle
import com.praszapps.fingertip.manager.FingertipAuthConfig
import com.praszapps.fingertip.manager.FingertipAuthenticationResult
import com.praszapps.fingertip.manager.FingertipAuthenticator

class SampleUsage : AppCompatActivity() {
    private val config = FingertipAuthConfig(AuthenticationStyle.FINGERPRINT_DIALOG, "Error")

    override fun onCreate(savedInstanceState: Bundle?) {
        super.onCreate(savedInstanceState)
        FingertipAuthenticator.INSTANCE.doAuth(config, object : FingertipAuthenticationResult {
            override fun onFingertipAuthSuccess() {

            }

            override fun onFingertipAuthFailed(errorString: String) {

            }
        })
    }
}